libtcmu_common.h

```diff
@@ -53,6 +53,8 @@ struct tcmulib_cmd;
 
 #define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
 
+#define VPD_MAX_WRITE_SAME_LENGTH 0xFFFF
```

Comment:
The ESXI's clone also will use the Write Same command some time, and for now the write same is a sync way, so just limit it to 64K, make sure the other commands won't cost too much time to wait Write Same to complete, this will be update later with enhancement patches.